packages/playground/src/data-boundary.tsx

```diff
@@ -0,0 +1,13 @@
+import { DataSource, useStream } from '@final-ui/react'
+import React, { PropsWithChildren } from 'react'
+import { Text, YStack } from 'tamagui'
+
+export function DataBoundary({ info, children }: PropsWithChildren<{ info: DataSource['info'] }>) {
```

Comment:
This package should live in core and be either provided out of the box or already wrapping Template.

Reply:
I would keep the header free from any status indicators, because users may utilise header buttons with their own UI. I suggest we come up with a nice non-intrusive UI to display this, maybe similar to React Frames Count overlay.